sys/net/if_bridge.c

```diff
@@ -2609,6 +2609,12 @@ bridge_input(struct ifnet *ifp, struct mbuf *m)
 			sc->sc_if_input(bifp, mc2);
 		}
 
+		/* Do not process none-local-targeted packets */
+  		if (!V_pfil_local_phys) {
+			m_freem(m);
```

Comment:
I'm recently working on https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=270478 and hit this logic.

I think the assumption `none-local-targeted packets` is wrong.

For multicast / broadcast packets, network stack (layer 2) does not know whether they are for us ( local-targeted ) or not, until upper protocol / applications claim they are.

So this change is probably wrong and breaks existing function.

I have immature thought that treat packets received from members as from bridge. That works like if_lagg. Then those bridge members will behave much like switch ports (no MAC) on real switches.